tests/kafkatest/services/connect.py

```diff
@@ -105,14 +104,13 @@ def set_external_configs(self, external_config_template_func):
         """
         self.external_config_template_func = external_config_template_func
 
-    def listening(self, node):
+    def rest_loaded(self, node):
```

Comment:
Why do we need to rename these methods? `listening` seems to be semantically correct; it's just that we're changing what the definition of "listening" means to actually be more than the port has been bound.

Given that the renaming doesn't really add much clarity, it may be better to keep the older name since that would reduce the changes. WDYT?

Reply:
Should we just change the call to `list_connector_plugins()` to use `_rest_with_retry` instead to fix the original failure in `KAFKA-8418`?

Reply:
@rhauch I am fine with keeping the old name if that is more clear and fewer changes.
@wicknicks we can do that as well, but putting a more strict check on started criteria will give us guarantee, that worker is actually started and serves REST. As an alternative - we can just replace `_rest`-> `_rest_with_retry` everywhere.

Reply:
This fix seems preferable because other rest calls might fail too, which we might not want to retry. Such as: `validate_config` and others.